src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -456,6 +461,9 @@ initCapabilities :: [Capability] -> EvalState
 initCapabilities cs = set (evalCapabilities . capGranted) cs def
 {-# INLINABLE initCapabilities #-}
 
+initModules :: ModuleCache -> EvalState -> EvalState
+initModules mcache = set (evalRefs . rsLoadedModules) mcache
+
```

Comment:
👍

Reply:
You can `INLINABLE` this call, as well. There's no harm in inlining this as a pure function

Reply:
We're pretty close to having nice benchmarks for these functions so I'd probably wait in order to avoid knee-jerk inlining ...